### ISSUE TITLE
Calculated field is updated, pubSub does not work

### ISSUE THREAD

**Issue by OllegK** [ideaportriga/nexus-bridge#116]
SWE_PROP_BC_NOTI_NEW_RECORD_DATA is not honoured

**Comment by OllegK** [comment]
It was reported that pubSub does not work when just a new value is picked from the pick applet, and the record is saved (AK) (maybe this issue could be reproduced only with the not immediate post change field? ). As far as I remember, the cp was skipped because it was received when the mvg/pick applet "opened" on the not-committed record.

**Comment by ostgals** [comment]
Actually yesterday I faced an issue that Siebel can send **several** SWE_PROP_BC_NOTI_NEW_RECORD_DATA notifications with 'cp' state set either to 0 and to 1 (within single notification session). It was an applet with complicated series of calculation while creating new record, triggering its 'commit pending' state back and forth several times.
So I have reworked my workaround to trigger UI updating only once with the latest 'cp' value after  SWE_PROP_BC_NOTI_END notification arrived.

**Comment by esneko** [comment]
@ostgals Could you create a PR with SWE_PROP_BC_NOTI_END workaround for the `nexus-bridge` or your implementation relies heavily on React?

**Comment by ostgals** [comment]
@esneko Yes, currently we have implemented this workaround as custom React hook. But I'll try to find time and to create PR to nexus-bridge next week.
